hw/ip/edn/doc/_index.md

```diff
@@ -118,6 +118,9 @@ Once firmware initialization is complete, it is important to exit this mode if t
 This is done by either *clearing* the `EDN_ENABLE` field or *clearing* the `BOOT_REQ_MODE` field in {{< regref "CTRL" >}} to halt the boot-time request state machine.
 Firmware must then wait for successful the shutdown of the state machine by polling the `REQ_MODE_SM_STS` field of the {{< regref "SUM_STS" >}} register.
 
+It should be noted that when in boot-time request mode, no status will be updated that is used for the software port operation.
+If some hang condition were to occur when in this mode, the main state machine debug register should be read to determine if a hang condition is present.
+
```

Comment:
@mwbranstad if it isn't done elsewhere already, could you add a snippet to boot-mode about how there is a finite amount of entropy, and it is the software's responsibility to switch to normal operation?

Reply:
@tjaychen just to clarify, when you say "finite amount of entropy", that means that what can be requested from EDN (we know that the ES can supply an infinite amount i suppose).

Reply:
ah yes that's right.

Reply:
Do you want the "when the command has completed"? The command may not complete, right? Also nit: typo in responsibility.